docs/application/dotnet/get-started/overview.md

```diff
@@ -1,33 +1,35 @@
 # Get Started with .NET Applications
 
-When you create your first Tizen .NET application, you can select from the following options that uses C# as the programming language:
+When you create your first Tizen .NET application, you can select from the following options that use C# as the programming language:
 
 -   [Wearable .NET application](wearable/first-app.md)
 
-    A wearable application is targeted to run on a smart watch. The application can be optimized for the smaller screen.
+    Applications targeted to run on smartwatches. It can be optimized for small screens.
 
 -   [Wearable XAML application](wearable/first-xaml-app.md)
 
-    A Wearable XAML application allows you to use XAML to define the visual contents of a page and work together with a C# code-behind file on a smart watch.
+    Applications that allows you to use XAML to define the visual contents of a page and work together with a C# code-behind file on a smartwatch.
 
 -   [TV .NET application](tv/first-app.md)
 
-    A TV application is targeted to run on a smart TV. The applications can be optimized for larger screens and remote control.
-    
+    Applications targeted to run on smart TVs. It can be optimized for large screens and remote controls.
```

Comment:
Suggestion.
I know your update in this PR is to shorten the sentences for the future work.
However, I'd like to suggest to focus on unifying the sentences and fixing the wrong singular/plural forms in the original sentences in this PR.

How about leaving the sentence shortening so that it can be checked together with the actual images when applying the UI?

Reply:
Sure, I will unify the sentences.